pkg/endpoint/policy.go

```diff
@@ -1008,8 +1008,8 @@ func (e *Endpoint) SetIdentity(identity *identityPkg.Identity) {
 
 	e.Consumable.Mutex.RLock()
 	e.getLogger().WithFields(logrus.Fields{
-		logfields.Identity: identity,
-		"consumable":       e.Consumable,
-	}).Debug("Set identity and consumable of EP")
+		logfields.Identity:       identity.StringID(),
+		logfields.IdentityLabels: identity.Labels.String(),
+	}).Info("Identity of endpoint changed")
```

Comment:
We log this to the EP log on line 999 but that is gated in an if. It might be good to also do a ep.LogStatus here so it shows up in the EP log directly. Note that those logs are printed to the debug log so it would duplicate but that should be ok.

Reply:
Oh, also, for the Info log here, it might be nice to include the old identity `StringID`, there is no clear context for it. Alternatively, since we're including IPs and podname in the standard log fields, it might be worth including the identity stringID in getLogger. This way, when filtering only for a specific endpointID you can see in the log when the identity switches. I'm not sure that this is super useful, though.